adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -248,8 +248,14 @@ private void handleEndpointConnection(final MqttEndpoint endpoint) {
                     LOG.debug("client [ID: {}] tries to publish on unsupported topic", endpoint.clientIdentifier());
                     close(endpoint);
                 }
+            });
 
+            endpoint.closeHandler(v -> {
+                LOG.debug("closing connection from client [{}]", endpoint.clientIdentifier());
```

Comment:
what about `client [{}] closes connection`? IMHO it gives a better indication who is actually closing the connection (the client).